frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -854,11 +839,11 @@ export default ({
         if (this.messages.length) {
           sinceHeight = Math.max(0, this.messages[0].height - limit)
         }
+
+        if (this.chatroomHasSwitchedFrom(chatRoomID)) return false
```

Comment:
I think this should be moved below line 844 as it was before, and also just `return` as before (to return `undefined`)